addons/sys_server/fnc_onGetRadioId.sqf

```diff
@@ -1,16 +1,18 @@
 /*
  * Author: ACRE2Team
- * SHORT DESCRIPTION
+ * This function is used to start initializing a radio for the intended player. The callback is used to complete to the process.
  *
  * Arguments:
- * 0: ARGUMENT ONE <TYPE>
- * 1: ARGUMENT TWO <TYPE>
+ * 0: Player's game object <OBJECT>
+ * 1: Radio base classname <TYPE>
+ * 2: CBA event name that is triggered when complete <STRING>
+ * 3: Optional: Replacement ID - Use this when copying data from another radio <STRING>
```

Comment:
` * 3: Replacement ID - Use this when copying data from another radio <STRING> (default: "")`